### DISCUSSION TITLE
Brief changelog for each release

### DISCUSSION THREAD

**Discussion by Mestizia** [nfdi4plants/ARCitect#270] [General]
Hi, 

I noticed that not all ARCitect releases come with a breakdown of the changes that are made to the codebase. The lack of clarity influences the process that data managers and data stewards have to go through when addressing problems that the users are encountering. 
Would it be possible in future releases to have a brief breakdown of the changes and what part of the tool are they affecting? It would make sense to somehow link swate and ARCitect changelogs as they are (in my experience) the major tools that the users interact with.

What is the general position of nfdi4plants regarding clarity in software releases?

**Comment by JonasLukasczyk** [comment]
This is related to #197 and going forward every release will have notes.
